src/components/your_grants/view_applicants/table/headers.tsx

```diff
@@ -1,31 +1,54 @@
 import { Flex, ResponsiveValue, Text } from '@chakra-ui/react';
 import React from 'react';
 
-function Headers() {
+function Headers({ isReviewer }:{ isReviewer : boolean; }) {
   const tableHeaders = [
```

Comment:
constants should be moved outside the function, and be SCREAMING_CASE

Reply:
this isn't a constant but a props